### PR TITLE
Add manual installation guide

### PR DESCRIPTION
**Pull request by AnjanaSenanayake**
This PR includes,
- [Add manual installation guide](https://github.com/wso2/open-healthcare-docs/pull/5/commits/25f2c20d069c623240076be398811814170bbd9f)
- [Remove update content and add update tool link](https://github.com/wso2/open-healthcare-docs/pull/5/commits/90ed9cb8046ac0a9fbcdfa26b9301638122ac8f2)

### CODE REVIEW

File: src/install-and-setup/manual.md [hunk 1]
```diff
@@ -1 +1,149 @@
...
+## Install WSO2 OH API Manager
+
+### Prerequisites
+1. Refer to [Product Compatibilities](#product-compatibilities) for Open Healthcare supported base product distributions.
+2. Download a Open Healthacare supported base distribution of [WSO2 API Manager](https://wso2.com/api-management/previous-releases/).
+3. Download the compatible WSO2 Open Healthcare APIM Accelerator. (Refer [Product Compatibilities](#product-compatibilities))
+
+### Installation Steps
+1. Extract WSO2 APIM product. Let's call it `<WSO2_APIM_HOME>`.
+2. Navigate to `<WSO2_APIM_HOME>/bin` directory and execute the update command based on the operating system to bring WSO2 API Manager up to date by running the Update Tool.
```
Comment (isuruuy429) [inline src/install-and-setup/manual.md]: Shall we provide the link to the Updates document? [1] Then we can get rid of providing how to update the base product. 
[1] - https://updates.docs.wso2.com/en/latest/updates/update-tool/

File: src/install-and-setup/manual.md [hunk 2]
```diff
@@ -1 +1,149 @@
...
+    - On Linux
+        ```sh
+        ./wso2update_linux --level x.y.z
+        ```
+            
+    - On Mac
+        ```sh
+        ./wso2update_darwin –level x.y.z
+        ```
+At first, you may need to run the relevant update command twice. First time you execute `./wso2update_<os>` updates the Update Client Tool. The subsequent `./wso2update_<os>` updates the product pack.
```
Comment (isuruuy429) [inline src/install-and-setup/manual.md:66]: Provide the link to the Wso2 Updates document. 
[1] - https://updates.docs.wso2.com/en/latest/updates/update-tool/

File: src/install-and-setup/manual.md (hunk 1 above)
Comment (AnjanaSenanayake) [reply src/install-and-setup/manual.md]: Update content removed mentioning link to update tool in [`90ed9cb`](https://github.com/wso2/open-healthcare-docs/pull/5/commits/90ed9cb8046ac0a9fbcdfa26b9301638122ac8f2)

File: src/install-and-setup/manual.md (hunk 2 above)
Comment (AnjanaSenanayake) [reply src/install-and-setup/manual.md:66]: Update content removed mentioning link to update tool in [`90ed9cb`](https://github.com/wso2/open-healthcare-docs/pull/5/commits/90ed9cb8046ac0a9fbcdfa26b9301638122ac8f2)

**Review by isuruuy429 — APPROVED** [review]

**Pull request merged**